formula_renames.json

```diff
@@ -48,6 +48,7 @@
   "gcc@6": "gcc",
   "geode": "apache-geode",
   "giflib5": "giflib@5",
+  "giflib@5": "giflib",
```

Comment:
I'd probably retire `"giflib5": "giflib@5",` to avoid the chaining but I believe @JCount said it sort of works.